arch/arm/core/aarch32/cortex_m/fault.c

```diff
@@ -136,8 +146,8 @@ static void fault_show(const z_arch_esf_t *esf, int fault)
 	PR_EXC("Fault! EXC #%d", fault);
 
 #if defined(CONFIG_ARMV7_M_ARMV8_M_MAINLINE)
-	PR_EXC("MMFSR: 0x%x, BFSR: 0x%x, UFSR: 0x%x",
-	       SCB_MMFSR, SCB_BFSR, SCB_UFSR);
+	PR_EXC("MMFSR: 0x%x, BFSR: 0x%x, UFSR: 0x%x", SCB_CFSR_MEMFAULTSR,
+	       SCB_CFSR_BUSFAULTSR, SCB_CFSR_USGFAULTSR);
```

Comment:
@gmarull Do you mean `SCB_CFSR_MEMFAULTSTR` (see the 2 latest characters SR vs STR)? 
This doesn't compile

Reply:
sorry I can see this has been fixed in main